app/pages/organization/organization-container.jsx

```diff
@@ -26,6 +26,7 @@ class OrganizationContainer extends React.Component {
       quoteObject: {}
     };
 
+    this.fetchAllOrganizationRoles = this.fetchAllOrganizationRoles.bind(this);
```

Comment:
I don't think this binding is necessary this class method isn't an event handler.

Reply:
without binding `fetchAllOrganizationRoles` in the `constructor` or alternatively and maybe preferable in the function itself when it's set as a `const` I get an error because the `organizationRoles` becomes undefined.

Reply:
I'm not sure why binding would affect default parameter in a class method. What happens if you instead remove the default and define it in inside the method?:
```
fetchAllOrganizationRoles(organization, organizationRoles, _page) {
  const organizationRoles = organizationRoles || [];
  const _page = _page || 1;
  // other stuff
}
```

Reply:
didn't realize I don't need to define the method as a `const` within itself, I think because I'm returning the promise, so just using `this.fetchAllOrganizationRoles` to continue to string promises works as desired. but not entirely sure this is the best solution, let me know what you think.